README.md

```diff
@@ -29,3 +29,126 @@ OpenEBS/project
 ```
 
 > Note: Developer should keep `integration_test` completely decoupled from the rest of the project packages.
+
+## Instantiation
+
+Developer has to instantiate CITF using `NewCITF(configFilePath)` which will initialize the CITF instance with all the required configurations. 
```

Comment:
Instead of `which will initialize the CITF instance` can we just us `which will initialize it`